src/magick-image.ts

```diff
@@ -143,6 +148,11 @@ export interface IMagickImage extends INativeInstance {
     modulate(brightness: Percentage): void;
     modulate(brightness: Percentage, saturation: Percentage): void;
     modulate(brightness: Percentage, saturation: Percentage, hue: Percentage): void;
+    negate(): void;
+    negate(onlyGrayScale: boolean): void;
```

Comment:
This is the same as Magick.NET but I think I want to remove the boolean overload and change that into a separate method? Maybe `negateGrayScale`?